autoscheduler/frontend/templates/index.html

```diff
@@ -13,6 +13,15 @@
                 background-color:#c2c2c2;
             }
         </style>
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-179924187-1"></script>
+        <script>
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+
+            gtag('config', 'UA-179924187-1');
+        </script>        
```

Comment:
There's some extra whitespace here
```suggestion
        </script>
```

Reply:
I am amazed you caught this